### PR TITLE
Add Tokyo 2018 meeting info and organizer contact info

### PR DESCRIPTION
**Pull request by moneyball**
@jonasschnelli @jnewbery can you guys take a look? Also I am not sure how you want to be represented on the web site as organizers so let me know if you'd like that changed.

### CODE REVIEW

**Comment by jnewbery** [comment]
Would be good to get this in soon (preferably on top of #3 , which has been open for some time)

File: index.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h1>CORE<span style="font-weight:100;">DEV</span> EVENTS</h1>
                 <hr>
                 <p>Invitation only developer events for hacking together on Bitcoin-Core and related projects.</p>
                 <p>__No talks. No politics. Just code.__</p>
-		<p><strong style="color: white;">NEXT EVENTS:<br />No event planed right now...</strong></p>
+		<p><strong style="color: white;">NEXT EVENTS:<br /><a href="tokyo_2018.html" style="color:white;">8th - 10th October 2018 - Tokyo</a></strong></p>
```
Comment (jnewbery) [inline index.html]: change `NEXT EVENTS` to `NEXT EVENT`

File: tokyo_2018.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
```
Comment (jnewbery) [inline tokyo_2018.html]: I tried to standardise on `<year>_<city>.html` in #3. Can I convince you to do the same here?

File: tokyo_2018.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                    <li class="active">
+                        <a class="page-scroll" href="pastevents.html">Past Events</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="index.html#about">Next Events</a>
+                    </li>
+                    <li>
+                        <a class="page-scroll" href="faq.html">FAQ</a>
+                    </li>
+                    <li>
```
Comment (jnewbery) [inline tokyo_2018.html]: I removed _Location_ and _Sponsors_ from the navigation bar in #3. I recommend you do the same here.

**Review by jnewbery — COMMENTED** [review]
Wording seems fine to me. A few nits inline.

**Comment by jnewbery** [comment]
Needs rebase :)

**Comment by moneyball** [comment]
rebased

**Comment by jnewbery** [comment]
LGTM. I've squashed all commits together.

Tested ACK 8578ac8bd2e4e4bcb538f5842bfa7ce227a7c948.

**Comment by jnewbery** [comment]
@jonasschnelli - please deploy to live site!

**Pull request merged**